src/components/magnet/field-lines.tsx

```diff
@@ -35,60 +36,72 @@ export default function FieldLines(props: IProps) {
         return;
       }
       const negEndOutsideX = magnetModel.flipped
-        ? magnet.x - (magnetModel.magnetLength / 2) - 1
-        : magnet.x + (magnetModel.magnetLength / 2) + 1;
+        ? magnet.x - (magnetModel.magnetLength  / 2) - 10
+        : magnet.x + (magnetModel.magnetLength  / 2) + 10;
       const negEndOuterX = magnetModel.flipped
-        ? magnet.x - (magnetModel.magnetLength / 2) + 1
-        : magnet.x + (magnetModel.magnetLength / 2) - 1;
+        ? magnet.x - (magnetModel.magnetLength  / 2) + 10
+        : magnet.x + (magnetModel.magnetLength  / 2) - 10;
```

Comment:
Oh also - when I made the field lines transparent I realized the `negEndOuterX` lines are now duplicates of the `negEndInnerX` lines. I would delete them